### PR TITLE
alerts and buttons

### PR DESCRIPTION
**Pull request by vijaytembugade**
Alerts Live preview: https://vijaytembugade.github.io/XenonUI-Component-Library/Components/Alerts/alerts.html
Buttons Live preview: https://vijaytembugade.github.io/XenonUI-Component-Library/Components/Buttons/buttons.html

No documentation added yet! Just components.

### CODE REVIEW

File: Components/Alerts/alerts.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    border-radius: 0.4rem;
+}
+
+@media only screen and (max-width: 768px) {
+    .error-alert{
+        background-color: var(--danger-color-shade);
+        color: var(--danger-color);
+        min-width: fit-content;
+        padding: 0.5rem;
+    }
```
Comment (mimansha-swarup) [inline Components/Alerts/alerts.css:38]: inside media query you don't need to write CSS of the class again
just mention changes that will work

eg:

.error-alert{
    background-color: var(--danger-color-shade);
    color: var(--danger-color);
    min-width: 25rem;
    padding: 0.5rem;
    border-radius: 0.4rem;
}
now In media query

@media only screen and (max-width: 768px) {
    .error-alert{
        min-width: fit-content;
    }
}

File: Components/Buttons/buttons.css [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+.btn-secondary-outline {
+  background-color: var(--light-color);
+  border: 2px solid var(--secondary-color);
+  color: var(--secondary-color);
+}
+.btn-secondary-outline:hover {
+  background: var(--secondary-color);
+  color: var(--light-color);
+}
+
```
Comment (mimansha-swarup) [inline Components/Buttons/buttons.css:98]: small suggestion
if you want your outline btn to have the same color as the parent then you can use 
background: inherit

**Review by mimansha-swarup — COMMENTED** [review]
these are few fixes you can do

File: Components/Alerts/alerts.css (hunk 1 above)
Comment (vijaytembugade) [reply Components/Alerts/alerts.css:38]: Okay. I will do this.

File: Components/Buttons/buttons.css (hunk 2 above)
Comment (vijaytembugade) [reply Components/Buttons/buttons.css:98]: Thanks. I will keep it in mind

**Review by srishti-maurya — APPROVED** [review]
the variants are great

**Comment by MarkVed17** [comment]
Those buttons look great! 
Especially on hovering over it.

**Review by mimansha-swarup — COMMENTED** [review]
good to go

**Review by horsemaker — COMMENTED** [review]
Great Work! So many variants.

**Pull request merged**